Better Vlach (Old Romanian) titles

hmlendea opened the pull request:
These are more accurate and fitting title localisations for the Vlach culture.
It's an update over the titles I've used in my mod for CK2 a while ago.

Notes:
 - Some of those titles are late medieval, as I wasn't able to find sources for how the titles were named in earlier periods. (I've only found sources that use the modern/modernised equivalent)
 - I've included some comments there for future reference, but if you want I can remove them. I still think they might be useful though

Explainations for some of the choices:
 - Changing "Român" to "Rumân": While they were internationally called "vlachs" back then, in their native tongue, they called themselves "rumâni" (singular "rumân"). The term later transformed into "român" which is still in use today. For some reason, "rumân" slowly changed meaning to "peasant", eventually becoming a term used exclusively for the poor workers / the lower class. Obviously, no one wanted to be called a "rumân" which was seen kind of like an insult. However, given CK3's time period, "rumân" is the correct term to use for the romanian/vlach people, as the negative connotations for that term have not yet developed.
 - "Țeară" is the Old Romanian form of the modern word "țară" which means country. It was also used for smaller regions (counties) and to some degree used today (only in name) in some places. Could theoretically be used for counties, but that would be overkill and only befitting for a few. However, it fits kingdoms very well as Țeară/Țară basically means a Tsardom (the domain of a Țar/Tsar which is the slavic word for a king or emperor, and comes from latin Caesar (emperor)). Real examples: Țeara Rumânească (Țara Românească), Țeara Moldovei (Țara Moldovei)
 - "Jude" being used for a republican baron: The "juzi" (plural for "jude") were basically the mayors of romanian cities in Wallachia. However, in Moldova, the mayor was called a "Șoltuz", which is a romanianised form of the transylvanian-saxon term "schulteiss" (mayor). However, I still think Jude should be the form used here, even if Moldova shares the same "vlach" culture in the game (unless, of course, it is of a foreign culture), as there was still a "jude" in Moldova as well. In fact, there were multiple "juzi" (2-4 of them), as they were subordinated to the "șoltuz", and were tasked with solving issues between local and foreign people to their city. Extra fact: The jude is called like that because he's "the one who passes judgement" in his city (judge = judecător in modern romanian).
 - Some (not all) of the female forms are educated guesses (I'm a native speaker), as there weren't any female romanian bans, voivodes, etc.
 - I know that the feudal county is named "Județ" and only the republican baron is named "Jude", but that's just how it was. The feudal count title "Ban" still fits as the bans were of two kinds: Ban de județ (simply, Ban), and Mare Ban (which I used for duke). My wild guess as to why counties were called județe is because there were probably many more towns in comparison to castles, and a town is ruled by a jude.

**Question:**
Did you guys find a way to localise council member titles? It used to be possible in CK2, and I do have the Old Romanian terms for those titles. I'd like to add them to this mod as well, if possible.

hmlendea commented:
I think the names are in a good shape now and they can be reviewed.
I will try adding more of them later though (e.g. for the theocracy government type)

Viridianus commented:
OK, first of all: such things are usually solved through our [sheet](https://docs.google.com/spreadsheets/d/1h_FkusqaFE_uAw0n1HUPFTqg4Ojq8DZqt-sR0eFagyo/edit), for all to see the reasoning.
I'll go through the details of what you did and be back with specifics, but it already seems that "Șoltuz" is a good fit for county-level majors.

As for council titles: they're localizable, but for technical reasons it's a different branch (Council-Tiltles-Alt as of now).

Viridianus commented on localization/english/culture/ctp_titles_l_english.yml:
```diff
@@ -1146,33 +1146,34 @@ empire_feudalclan_serbian:0 "Carstvo"
+county_feudalclan_vlach:0 "Județ"
+county_tribal_vlach:0 "Cnezat"
+county_feudalclan_vlach_independent:0 "Cnezat"  
+duke_feudalclan_male_vlach:0 "Banat"
```
Clearly a mistake here: Banat should've been duchy or county - not a person.

hmlendea replied:
It is, thank you

Viridianus commented on localization/english/culture/ctp_titles_l_english.yml:
```diff
@@ -1146,33 +1146,34 @@ empire_feudalclan_serbian:0 "Carstvo"
+baron_republic_male_vlach:0 "Jude" # Used in Wallachia. In Moldova it was "Șoltuz" (romanianised variant of the transylvanian saxon "schulteiss" (mayor))
+baron_feudalclan_female_vlach:0 "Castelană"
+baron_republic_female_vlach:0 "Jude" # I'm not aware of any feminine variant but the original one is pretty gender-neutral
+barony_feudalclan_vlach:0 "Cetate"
```
This one is also kinda suspect - ain't it "civitas", which is whatever but not feudal?

hmlendea replied:
Cetate is a fortress / stronghold / castle

Viridianus commented on localization/english/culture/cultures_l_english.yml:
```diff
@@ -374,8 +374,8 @@
- vlach:0 "Român"
- vlach_collective_noun:1 "Român"
+ vlach:0 "Rumân" # Later transformed into "Român" as "rumân" eventually became a term exclusive for the lower class
+ vlach_collective_noun:1 "Rumân"
```
It should be plural - won't it be Rumăni?

hmlendea replied:
Rumâni*

Or "Rumîni" as someone suggested on the sheets. I've added to the discussion there

Viridianus reviewed (changes requested):
I left several comments, but the whole change is clearly done, well, in haste.

Charanraja123 commented:
I think i agree with @Viridianus in saying its better if these things go through the sheet. The main reason of all, Someone there has entered something for this, and we owe to them to tell them why it might incorrect and give them a chance to respond. 

We are doing the Council Titles. We are still testing things there from a technical side. Also, we already have some council titles for Vlach, please check the sheets for those too to see if you agree with them.

We haven't started theocratic as we are still collecting data (and modding in other stuff). You can enter your stuff in the religious titles page in the Google sheets 

As you would have guessed already. GitHub is not where we decide title name changes, This is more of a technical discussion area. The Sheets and the forum are where all the discussions regarding Titles happen.
Sorry for the inconvenience .

hmlendea reviewed (commented):
Thanks for the feedback.

Yes, mostly I created this while still working on it (I always do my work directly in a git repo) and opened this PR early so that it can be reviewed and get suggestions and corrections - which was the case indeed as there were a few mistakes that I didn't notice.

I didn't know about the sheets though. I found this repository while searching for it on Google, as I'm not a user of the paradox forums, and there wasn't a readme to mention it
I will add the suggestions to the sheet later

Charanraja123 commented:
Sorry that is our bad.
We really didn't expect anyone who didn't use Paradox Forum to visit us here. 
I'll update the readme. Thanks.

hmlendea commented:
Ah, well, I think mine was just an edge case :P I think most people reach this repo through more normal means

Viridianus commented:
Just in case: after we finish the discussion on sheet, it's, obviously, on you, @hmlendea, to implement it in your branch (for all the four languages, since you've started) and notify us about it. I will _not_ Merge+Master-Fix it, and not only because Master-Fix is now somewhat split from master.

hmlendea commented:
Of course, that goes without saying.

But I think it will be bit before we're comfortable implementing them.
The discussions on the sheets have been very constructive and while doing additional research I've found new details and more sources on different situations for when the ttiles were used and most importantly in what period it happened.

Viridianus commented:
Merge master branch into your own (or, better even, rebase).

Viridianus commented:
Well, before I even start a review - the empire's practice is not in line with the rest where it's usually the same title for all the four governments (and, consequently, government is not marked in the name); likewise, kingdom is usually not different for feudals and tribals. It might be the same in our master branch currently, of course, too, but it will probably be changed at some point unless you have some reservations about it.

hmlendea commented:
It's not yet ready for review as I still have to make sure there isn't anything left to change, and more importantly to test it in-game.

I will make kingdom and empire work across all governments as the same name should indeed fit all of them.

Viridianus commented:
Good (just remember not to overwrite kingdom_republic accidentally). Also, we'd be honored for you to join us at [Discord](https://discord.gg/vSxPUnZ)

Viridianus commented:
Come on, didn't I specifically mention not overriding kingdom-level republics? Please delete republic_government from all the kingdom-level non-republic titles (that includes the main kings as well as Muslim kings). But really, thx for the effort ;)

Viridianus reviewed (approved):
Aside from republic_government for kingdoms, there's another technical failure. While sheet states princes of empire-level default to princes of kingdom-level, code-wise they must be written separately. We can fix that ourselves, of course, but it would be cool if we merged it already with all princes.
Otherwise, everything's great.

hmlendea commented:
> Come on, didn't I specifically mention not overriding kingdom-level republics? Please delete republic_government from all the kingdom-level non-republic titles (that includes the main kings as well as Muslim kings). But really, thx for the effort ;)

Hah, I completely misread that, lol! Sorry

hmlendea commented:
Ok, I think that's it now.
I've tested the titles in-game and fixed the issues that I've found and now they seem to be working ok.
Please take another look to see if I missed something or if I didn't adhered to conventions.

**Note:**
I did encounter a strange behaviour as the Goidelic group title for sultanate and republic-kingdom were used instead of vlach, until I changed the priority value.
I'm not sure what was going on there since the flavourisations for those goidelic titles was set correctly with `culture_groups = { goidelic_group }` and I don't understand why it was used for `vlach`.

Viridianus reviewed (approved):
Yep, you're all set now.

Charanraja123 commented:
@hmlendea Was this a one-off thing or is it reproducible? 
As you say everything looks in order for me too. I have no idea on why that would happen, especially with that priority change. 

What was your original priority values for those both?

Charanraja123 commented:
Well, If you still have any error showing up in the release version, let us know. Vlach has been added to the steam release version.

Off-Topic: Also we would like to talk to you about our place names research. Please tell us when you will be free. At least one of us will be available on Discord at almost all times.
